src/Parser/RestoDruid/Modules/Features/StatWeights.js

```diff
@@ -0,0 +1,316 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+import HIT_TYPES from 'Parser/Core/HIT_TYPES';
+import { formatNumber } from 'common/format';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import HealingValue from 'Parser/Core/Modules/HealingValue';
+import CritEffectBonus from 'Parser/Core/Modules/Helpers/CritEffectBonus';
+
+import { getSpellInfo } from '../../SpellInfo';
+
+const DEBUG = false;
+
+const ARMOR_INT_MULTIPLIER = 1.05; // 5% int bonus from wearing all leather means each new point of int worth 1.05 vs character sheet int
+
+/*
+ * Resto Druid Stat Weights Methodology
+ *
+ * Approach -
+ * This module generates the players stat weights using the actual logged events. We keep a listing of all the player's healing spells
+ * along with which stats those spells scales with, and for each stat a heal scales with we do some simple math to find out
+ * how much more it would have healed if the player had one more of that stat. We compare the total healing increases
+ * caused by seperately increasing each stat by one in order to generate weights.
+ *
+ * Overheal -
+ * The toughest problem to face when generating healing stat weights is what to do about overhealing. Unfortunately, there is
+ * no one correct answer. A main issue is what I call The "Would've" Problem. We can say "an extra bit of healing here doesn't
+ * matter because it *would've* just caused the next heal to overheal", but how far can we take this logic? My approach is that
+ * it doesn't only matter how much is healed, but also how quickly. We'll count all healing that doesn't overheal even if we
+ * theorize that it might lead to overhealing. We're more strict on heals that overheal: they're disregarded entirely even
+ * if they're only partial overhealing. This is because a spell that partially overheals will do the same amount of effective
+ * healing regardless of the raw strength of the heal. This approach can cause some possible weirdness. For example, consider
+ * a situation where there is a heal for 500 (0 overheal) followed immediately by a heal for 400 (+ 100 overheal). 500 of this
+ * healing would be counted towards stat weights. Now consider a situation where one heal does 900 (+ 100 overheal). Functionally,
+ * this situation is identical, but in this case none of the healing is counted towards stat weights. Counting all effective
+ * healing, even heals that partially overheal, fixes this issue and is overall a valid approach, but still not one I will be taking.
+ * I think an advantage of disregarding all partial overheals is that it gives "top off" heals an effective lower weight than
+ * "life saver" heals. Still, this is a decision I will revisit.
+ *
+ * Stat Tracking -
```

Comment:
I'll make a few more updates to make the StatTracker fit better. Because of the manual handling of Concordance and 2t19, I'm satisfied enough with this module even without StatTracker.